scripts/cl_ext.h.mako

```diff
@@ -162,7 +162,16 @@ orderedExtensions = [
 
 def getExtensionSortKey(item):
     name = item.get('name')
-    return orderedExtensions.index(name)
+    try:
+        index = orderedExtensions.index(name)
+    except ValueError:
+        if name.startswith('cl_khr'):
+            index = 10000
```

Comment:
What's the behaviour when the same index is returned multiple times? Are definitions for extensions ordered following the order in which they appear in the XML?

Reply:
I had to look this up.  The sort is stable if they have the same key:
https://docs.python.org/3/howto/sorting.html#sort-stability-and-complex-sorts

However, the sort key is actually a tuple consisting of the index and the name, so the end result should be that the new extensions are ordered after the older manually ordered extensions and alphabetized.  This is the order I'd like to see for all extensions eventually - see `getExtensionSortKey_ideal` below.